From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 6 Dec 2014 22:39:09 +0000
Subject: [PATCH 5/9] proc connector: Delete spurious memset in
 proc_exit_connector()

Upstream commit e727ca82e0e9 ("proc connector: fix info leaks")
changed many functions that don't exist in 2.6.32.y.  When it was
cherry-picked into 2.6.32.61, one extra memset() calls was inserted
into proc_exit_connector().  This results in clearing the cpu
field of exit events.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 drivers/connector/cn_proc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/connector/cn_proc.c b/drivers/connector/cn_proc.c
index 3603599..551ea92 100644
--- a/drivers/connector/cn_proc.c
+++ b/drivers/connector/cn_proc.c
@@ -187,7 +187,6 @@ void proc_exit_connector(struct task_struct *task)
 	memset(&ev->event_data, 0, sizeof(ev->event_data));
 	get_seq(&msg->seq, &ev->cpu);
 	ktime_get_ts(&ts); /* get high res monotonic timestamp */
-	memset(&ev->event_data, 0, sizeof(ev->event_data));
 	put_unaligned(timespec_to_ns(&ts), (__u64 *)&ev->timestamp_ns);
 	ev->what = PROC_EVENT_EXIT;
 	ev->event_data.exit.process_pid = task->pid;
